netmanager/src/views/components/Sites/SitesTable.js

```diff
@@ -23,196 +23,190 @@ import HorizontalLoader from 'views/components/HorizontalLoader/HorizontalLoader
+          },
+          {
+            title: 'Country',
+            field: 'country',
+            render: renderCell('country'),
+            cellStyle: { fontFamily: 'Open Sans' }
+          },
+          {
+            title: 'District',
+            field: 'district',
+            render: renderCell('district'),
+            cellStyle: { fontFamily: 'Open Sans' }
+          },
+          {
+            title: 'Region',
+            field: 'region',
+            render: renderCell('region'),
+            cellStyle: { fontFamily: 'Open Sans' }
+          },
+          {
+            title: 'Actions',
+            render: (rowData) => (
+              <div>
+                <Tooltip title="Delete">
+                  <DeleteIcon
+                    // className={"hover-red"}
+                    style={{
+                      margin: '0 5px',
+                      cursor: 'not-allowed',
+                      color: 'grey'
+                    }}
+                    // disable deletion for now
+                    // onClick={(event) => {
+                    //   event.stopPropagation();
+                    //   setDelState({
+                    //     open: true,
+                    //     name: rowData.name || rowData.description,
+                    //     id: rowData._id,
+                    //   });
+                    // }}
+                  />
+                </Tooltip>
+              </div>
+            )
+          }
+        ]}
+        onRowClick={(event, data) => {
+          event.preventDefault();
+          clearSiteDetails();
+          history.push(`/sites/${data._id}`);
+        }}
+        data={sites}
+        options={{
+          search: true,
+          exportButton: true,
+          searchFieldAlignment: 'right',
+          showTitle: true,
+          searchFieldStyle: {
+            fontFamily: 'Open Sans'
+          },
+          headerStyle: {
+            fontFamily: 'Open Sans',
+            fontSize: 16,
+            fontWeight: 600
+          },
+          exportCsv: (columns, data) => {
+            const fields = [
+              'name',
+              'description',
+              'generated_name',
+              'latitude',
+              'longitude',
+              'country',
+              'region',
+              'district',
+              'city',
+              'county',
+              'sub_county',
+              'parish',
+              'street',
+              'formatted_name',
+              'altitude',
+              'greenness',
+              'landform_90',
+              'landform_270',
+              'aspect',
+              'distance_to_nearest_road',
+              'distance_to_nearest_primary_road',
+              'distance_to_nearest_tertiary_road',
+              'distance_to_nearest_unclassified_road',
+              'distance_to_nearest_residential_road',
+              'distance_to_nearest_secondary_road',
+              'distance_to_kampala_center',
```

Comment:
Hi @BenjaminSsempala , I notice that we still have "`distance_to_kampala`" in Netmanager. I thought this was already done.